docs/download_files.md

```diff
@@ -21,42 +22,107 @@ The files shown below will be included with each library (example shown for a li
+This includes a summary of the types of libraries (e.g., single-cell, single-nuclei, with CITE-seq) and sample diagnoses included in the object, as well as UMAP visualizations highlighting each library.
+
+Every download also includes the individual [QC report](#qc-report) and, if applicable, [cell type annotation reports](#cell-type-report) for each library included in the merged object.
+
+<!--TODO: Update these images to use new folder titles-->
+
+#### Download folder structure for `SingleCellExperiment` merged downloads:
+![project download folder](images/merged-project-download-folder.png){width="600"}
+
+#### Download folder structure for `AnnData` merged downloads:
+![project download folder](images/merged-anndata-project-download-folder.png){width="600"}
+
+#### Download folder structure for `AnnData` merged downloads with CITE-seq (ADT) data:
+![project download folder](images/merged-anndata-project-citeseq-download-folder.png){width="600"}
+
+
+## Custom datasets
+
+<!--TODO: Fill in this section with information about creating and downloading custom datasets-->
+
+## Portal-wide downloads
+
+<!--TODO: Fill in information about portal-wide downloads and provide screenshots of all file structure downloads -->
+
+### Metadata-only downloads
+
+The Portal-wide metadata download is a single TSV file containing the metadata for all samples with associated single-cell RNA-seq, single-nuclei RNA-seq, or spatial transcriptomics data available on the Portal. 
+A table describing all columns included in the file can be found in the [metadata section below](#metadata).
+
+### `SingleCellExperiment` downloads
+
+#### Download folder structure for individual sample downloads:
+![sample download folder](images/sample-download-folder.png){width="600"}
 
-### Download folder structure for individual sample downloads:
+### `AnnData` downloads
+
+#### Download folder structure for individual sample downloads:
 ![sample download folder](images/anndata-sample-download-folder.png){width="600"}
 
-### Download folder structure for individual sample downloads with CITE-seq (ADT) data:
+#### Download folder structure for individual sample downloads with CITE-seq (ADT) data:
 ![sample download folder](images/anndata-sample-citeseq-download-folder.png){width="600"}
 
 If downloading a sample that contains a CITE-seq library as an `AnnData` object (`.h5ad` file), the quantified CITE-seq expression data is included as a separate file with the suffix `_adt.h5ad`.
 
+<!--TODO: 
+move spatial section 
+merged objects
+-->
+
 ## Gene expression data
 
 Single-cell or single-nuclei gene expression data is provided as either [`SingleCellExperiment` objects (`.rds` files)](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html) or [`AnnData` objects (`.h5ad` files)](https://anndata.readthedocs.io/en/latest/index.html).
```

Comment:
i just noticed this while scrolling through, do we want to update any of these OSCA links (more generally?)